### PR TITLE
Remove JP2 Generation

### LINKED ISSUE #1854: Periodic test failures accessing helper from decorator
```
Failures:

  1) EventGenerator::GeoblacklightEventGenerator#record_member_updated with a record in a completed state publishes a persistent JSON updated message with geoblacklight document
     Failure/Error: h.visibility_badge(visibility)
     
     NoMethodError:
       undefined method `visibility_badge' for #<#<Class:0x000000001c419fa8>:0x000000001e84f9b8>
     # ./app/decorators/valkyrie/resource_decorator.rb:48:in `block in visibility'
     # ./app/decorators/valkyrie/resource_decorator.rb:47:in `map'
     # ./app/decorators/valkyrie/resource_decorator.rb:47:in `visibility'
     # ./app/decorators/application_decorator.rb:39:in `[]'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:27:in `block in direct_attributes'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:26:in `map'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:26:in `direct_attributes'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:18:in `merged_attributes'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:13:in `merged_attributes_object'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:35:in `block in build_simple_attributes'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:34:in `each'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:34:in `build_simple_attributes'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:14:in `build'
     # ./app/services/geo_discovery/document_builder/composite_builder.rb:15:in `block in build'
     # ./app/services/geo_discovery/document_builder/composite_builder.rb:14:in `each'
     # ./app/services/geo_discovery/document_builder/composite_builder.rb:14:in `build'
     # ./app/services/geo_discovery/document_builder.rb:33:in `initialize'
     # ./spec/services/event_generator/geoblacklight_event_generator_spec.rb:72:in `new'
     # ./spec/services/event_generator/geoblacklight_event_generator_spec.rb:72:in `block (4 levels) in <top (required)>'

  2) EventGenerator::GeoblacklightEventGenerator#record_updated with a record in a completed state publishes a persistent JSON updated message with geoblacklight document
     Failure/Error: h.visibility_badge(visibility)
     
     NoMethodError:
       undefined method `visibility_badge' for #<#<Class:0x000000001c419fa8>:0x000000001e84f9b8>
     # ./app/decorators/valkyrie/resource_decorator.rb:48:in `block in visibility'
     # ./app/decorators/valkyrie/resource_decorator.rb:47:in `map'
     # ./app/decorators/valkyrie/resource_decorator.rb:47:in `visibility'
     # ./app/decorators/application_decorator.rb:39:in `[]'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:27:in `block in direct_attributes'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:26:in `map'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:26:in `direct_attributes'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:18:in `merged_attributes'
     # ./app/decorators/geoblacklight_metadata_decorator.rb:13:in `merged_attributes_object'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:35:in `block in build_simple_attributes'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:34:in `each'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:34:in `build_simple_attributes'
     # ./app/services/geo_discovery/document_builder/basic_metadata_builder.rb:14:in `build'
     # ./app/services/geo_discovery/document_builder/composite_builder.rb:15:in `block in build'
     # ./app/services/geo_discovery/document_builder/composite_builder.rb:14:in `each'
     # ./app/services/geo_discovery/document_builder/composite_builder.rb:14:in `build'
     # ./app/services/geo_discovery/document_builder.rb:33:in `initialize'
     # ./spec/services/event_generator/geoblacklight_event_generator_spec.rb:29:in `new'
     # ./spec/services/event_generator/geoblacklight_event_generator_spec.rb:29:in `block (4 levels) in <top (required)>'

Finished in 7 minutes 24 seconds (files took 16.35 seconds to load)
2792 examples, 2 failures, 35 pending

Failed examples:

rspec ./spec/services/event_generator/geoblacklight_event_generator_spec.rb:71 # EventGenerator::GeoblacklightEventGenerator#record_member_updated with a record in a completed state publishes a persistent JSON updated message with geoblacklight document
rspec ./spec/services/event_generator/geoblacklight_event_generator_spec.rb:28 # EventGenerator::GeoblacklightEventGenerator#record_updated with a record in a completed state publishes a persistent JSON updated message with geoblacklight document

Randomized with seed 14954
```

### LINKED ISSUE #3467: Intermittently failing spec in RasterResourceDecorator
https://circleci.com/gh/pulibrary/figgy/21801
```
Failures:

  1) RasterResourceDecorator exposes markup for rendered coverage
     Failure/Error: h.bbox_display(display_coverage)
     
     NoMethodError:
       undefined method `bbox_display' for #<#<Class:0x0000560843db1720>:0x0000560843a3daf8>
       Did you mean?  display
     # ./app/resources/raster_resources/raster_resource_decorator.rb:70:in `rendered_coverage'
     # ./spec/resources/raster_resources/raster_resource_decorator_spec.rb:29:in `block (2 levels) in <top (required)>'
```

### LINKED ISSUE #3940: Stop making JPEG2000 derivatives
We should stop making JPEG2000 derivatives now that we're using pyramidal TIFFs. Include an ADR.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #3940 

Also hopefully closes #1854 and closes #3467

### CODE REVIEW

File: spec/services/manifest_builder_spec.rb [hunk 1]
```diff
@@ -439,18 +439,6 @@ def logical_structure(file_set_id)
       end
     end
 
-    context "when in staging and pyramidals are disabled" do
```
Comment (hackmastera) [inline spec/services/manifest_builder_spec.rb:442]: What happens now in this case?

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -28,10 +28,8 @@ module Features
   # Prevent leaking view contexts between tests
   # see https://github.com/drapergem/draper/issues/814
   # see https://github.com/drapergem/draper/issues/655
-  [:decorator, :controller, :mailer].each do |type|
-    config.before(:each, type: type) { Draper::ViewContext.clear! }
-    config.after(:each, type: type) { Draper::ViewContext.clear! }
-  end
+  config.before { Draper::ViewContext.clear! }
```
Comment (hackmastera) [inline spec/rails_helper.rb:31]: Maybe this will fix #1854 and #3467???

File: app/exporters/hathi/content_package.rb [hunk 3]
```diff
@@ -139,8 +132,20 @@ def image_file
...
+      def image_filename
+        "#{@basename}.jp2"
+      end
+
+      def path_to_file
+        @path_to_file ||= Pathname.new(jp2_derivative.path)
+      end
+
+      def jp2_derivative
+        @jp2_derivative ||= JP2Creator.new(filename: original_file_path.to_s).generate
```
Comment (hackmastera) [inline app/exporters/hathi/content_package.rb:144]: What will be the impact on the creation of these packages now? I assume they will take much longer to create? I.e. should we ask @cwulfman to test it with this branch?

File: spec/rails_helper.rb (hunk 2 above)
Comment (tpendragon) [reply spec/rails_helper.rb:31]: :pray: This was my hope.

File: spec/services/manifest_builder_spec.rb (hunk 1 above)
Comment (tpendragon) [reply spec/services/manifest_builder_spec.rb:442]: Nothing. There's no such thing as disabling pyramidal access anymore.

File: app/exporters/hathi/content_package.rb (hunk 3 above)
Comment (tpendragon) [reply app/exporters/hathi/content_package.rb:144]: Yes, they'll definitely take longer to create, but I also don't think we're actively generating these right now. I'd be disinclined to hold this up for that, but @cwulfman should definitely try stuff out again if he's still working on this.

Comment (tpendragon) [reply app/exporters/hathi/content_package.rb:144]: Yes, they'll definitely take longer to create, but I also don't think we're actively generating these right now. I'd be disinclined to hold this up for that, but @cwulfman should definitely try stuff out again if he's still working on this.

Comment (hackmastera) [reply app/exporters/hathi/content_package.rb:144]: 👍

File: spec/rails_helper.rb (hunk 2 above)
Comment (hackmastera) [reply spec/rails_helper.rb:31]: I'm so hopeful i'm gonna add it to the PR description to auto-close them.

Comment (eliotjordan) [reply spec/rails_helper.rb:31]: I'm hopeful as well: https://github.com/drapergem/draper/issues/523

**Pull request merged**